### ISSUE TITLE
ZeroOrMore typing rejects strings

### ISSUE THREAD

**Issue by ydirson** [pyparsing/pyparsing#456]
When typechecking code with a `ZeroOrMore` literal (e.g. `ZeroOrMore(".")`), my complains:

```
error: Argument 1 to "ZeroOrMore" has incompatible type "str"; expected "ParserElement"  [arg-type]
```

`ZeroOrMore` is annotated with ` expr: ParserElement`, shouldn't that be `Union[ParserElement, str]` ?

**Comment by ptmcg** [comment]
Yes, thanks! Will be corrected in 3.0.10

**Comment by ptmcg** [comment]
Released in 3.1.0a1
